mazerunner-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXGetJavaDataScenario.java

```diff
@@ -0,0 +1,39 @@
+package com.bugsnag.android.mazerunner.scenarios;
+
+import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.Configuration;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+
+import androidx.annotation.NonNull;
+
+public class CXXGetJavaDataScenario extends Scenario {
+
+    static {
+        System.loadLibrary("bugsnag-ndk");
+        System.loadLibrary("monochrome");
+        System.loadLibrary("entrypoint");
+    }
+
+    public native void activate();
+
+    /**
```

Comment:
Unfortunately these are necessary due to the linter I believe.

Reply:
Because it whats any JavaDoc?  Fair enough, but in that case perhaps put a simple sentence in there for the actual doc - just so it doesn't look an omission to the untrained (i.e. my) eye.